store/tikv/region_request.go

```diff
@@ -271,10 +278,17 @@ func (s *RegionRequestSender) SendReqCtx(
 			logutil.Logger(bo.GetCtx()).Warn("retry get ", zap.Uint64("region = ", regionID.GetID()), zap.Int("times = ", tryTimes))
 		}
 
+		var lastStoreID uint64
```

Comment:
Why we should maintain `lastStoreID`?

Reply:
Have removed it, we won't need it anymore.